<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataLogDlg</class>
 <widget class="QWidget" name="DataLogDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据日志</string>
  </property>
  <widget class="QTableWidget" name="DataListTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="CurrentPageLabel">
      <property name="text">
       <string>第100页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TotalPageabel">
      <property name="text">
       <string>共10000页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PageToBeginBtn">
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PageDownBtn">
      <property name="text">
       <string>前一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PageUpBtn">
      <property name="text">
       <string>后一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PageToEndBtn">
      <property name="text">
       <string>末页</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
